episodes/designing-a-light-microscopy-experiment.md

```diff
@@ -0,0 +1,402 @@
+features (like the nucleus or cell membrane). These labels are excited by a 
+specific wavelength of light (the excitation wavelength), and then emit light of 
+a longer wavelength (the emission wavelength). By illuminating the sample with 
+light of the excitation wavelength, then detecting light of the emission 
+wavelength, a fluorescence microscope can image the biological structures that 
+the label is bound to.
+
+![The image above is a fluorescence microscopy image of some LLC-PK1 cells 
+(ZEISS Microscopy, [CC BY 2.0](https://creativecommons.org/licenses/by/2.0), 
+via [Wikimedia Commons
+](https://commons.wikimedia.org/wiki/File:Mitotic_LLC-PK1_cells,_fluorescence_microscopy_(23700644352).jpg)
+)](fig/fluorescence-example.jpg){alt="Fluorescence microscopy image of some 
+LLC-PK1 cells" width='70%'}
+
+The main issue with widefield microscopes is that, due to the whole sample being 
+illuminated, it can produce rather blurry images (especially for thicker 
+samples). For example, in widefield fluorescence microscopy, fluorophores 
+throughout the entire sample will be emitting light. This means that the 
+resulting image will represent light from the focal plane, and also from 
+out-of-focus planes above and below it. This can result in details being 
+obscured by out-of-focus light.
+
+### Confocal
+
+Confocal microscopes are capable of 'optical sectioning' meaning they can detect 
+light only from the plane of focus and exclude light from out-of-focus planes. 
+This is extremely useful, especially for thick samples, to achieve clear images 
+where details aren't obscured by out-of-focus light.
+
+There are many different types of confocal microscopes, but one of the most 
+common is: laser scanning confocal microscopes. These are also known as point 
+scanning confocal microscopes. In this microscope, a laser is focused onto a 
+single point of the specimen at a time (rather than illuminating the entire 
+sample). The point of illumination is then moved across the sample in a raster 
+pattern (i.e. line by line), with light detected point by point. This gradually 
+builds up the final image.
+
+Confocal microscopes are typically used for fluorescence microscopy, especially 
+when it is necessary to image the shape of structures in full 3D. Bear in mind 
+that their advantages don't come without other disadvantages! For example, 
+confocal microscopes will generally be more complex to use and slower to acquire 
+images than a widefield microscope. Although there are other types of confocal 
+that are faster than laser scanning systems, such as 'spinning disc confocal 
+microscopes'.
+
+![The image above is Napari's Kidney (3D + 3Ch) sample image. This was acquired 
+with confocal fluorescence microscopy.](fig/confocal-example.png){alt="Screenshot 
+of Napari's Kidney (3D + 3Ch) sample image" width='60%'}
+
+### Super-resolution
+
+As there are so many [different 'super-resolution' methods
```

Comment:
I would move this first sentence to the end of the paragraph. And maybe put some examples of scale as to what super-resolution can do and what is the theoretical limit that it is breaking in terms of distance.

Reply:
I've added some values for the diffraction limit + maximum resolutions of standard vs super-resolution microscopes